### DISCUSSION TITLE
Doris v3 Helm Chart

### DISCUSSION THREAD

**Discussion by adeel-ansari** [apache/doris#45006] [A - General / Q&A] [answered]
Hi, do you know when the Helm chart for Doris v3 will be released?  I believe the latest helm supports v2.1.2?

Thanks.

**Comment by adeel-ansari** [comment]
Just following up on this.  I don't think the docker image for v3.0.3 is out yet either.  I'm checking here: https://hub.docker.com/r/apache/doris/tags

**Comment by ixzc** [comment, marked as answer]
you can use this 
https://hub.docker.com/r/selectdb/doris.be-ubuntu/tags
https://hub.docker.com/r/selectdb/doris.fe-ubuntu/tags
